README.md

```diff
@@ -1,22 +1,42 @@
-## Istio Installer
-This image is used by the rancher istio dev-chart to intall istio with various configurations 
+# Istio Installer
+This image is used by the [rancher-istio chart](https://github.com/rancher/charts "rancher-istio chart") to intall istio with various configurations.
 
-Scripts include install, upgrade and uninstall. 
+This installer uses shell scripts to run istioctl install, upgrade and uninstall commands. 
 
 ## Build
-Image is built when a new tag is created and pushed to repository.
+The istio-installer image is built when a new tag is created and pushed to repository.
 
+####  How to tag
+The istio installer uses the following version format for tags: `<x.x.x>-rancher<x>`
+
+The `<x.x.x>` should be set the version of istio the installer is going to be used for. This can be found in the Dockerfile as the value for the `ENV ISTIO_VERSION`
+
+The `rancher<x>` is the build number. This number starts at 1 and increases with each re-tag with the same istio version.`
+
+After all of the changes have been merged into the master branch, fetch a fresh copy of the master branch to your local machine. From there, you can run the following commands to tag and push the tag to the repo
 ```
 git tag <x.x.x>-rancher<x>
 git push <istio_installer_remote_branch> <x.x.x>-rancher<x>
 ```
+Once the tag is pushed to github, drone will build the docker image and push it to the [rancher/istio-installer](https://hub.docker.com/r/rancher/istio-installer "rancher/istio-installer") docker hub repository.
+
+**An example tag for a new istio version and push may look like this:**
+```
+git tag 1.9.6-rancher1
+git push upstream 1.9.6-rancher1
+```
+If you need to re-tag the same version, that may look like this:
+```
+git tag 1.9.6-rancher2
+git push upstream 1.9.6-rancher2
+```
+## Architecture
+User `istioctl` and `nginx` to complete install, upgrade and delete comands for the Istio service mesh. 
```

Comment:
Not sure what this is supposed to say but it doesn't read correctly. User -> Use maybe?

Reply:
Fixed in most recent commit - it is now "uses"